src/helpers/client/safari-client.ts

```diff
@@ -39,6 +39,11 @@ export interface PromiseExecutor {
   reject: (error: any) => void;
 }
 
+/**
+ * Global keep alive interval to prevent recursion.
+ */
+let keepAliveInterval: NodeJS.Timeout;
```

Comment:
Is this right? Seems strange to see a NodeJS timeout model when this should come from the browser?

Reply:
Sorry. Forgot this was browser based. This incorrect. Let me find the right type. I have it somewhere else.

Reply:
Use 

`ReturnType<typeof setTimeout> `

Reply:
```suggestion
let keepAliveInterval: ReturnType<typeof setTimeout>;
```

Reply:
Done, just changed the type